tests/testerina-integration-test/src/test/resources/command-outputs/unix/ModuleExecutionFlowTest-test-listener-shutdown.txt

```diff
@@ -0,0 +1,29 @@
+Compiling source
+	wso2/moduleExecutionFlow:0.0.0
+
+Running Tests
+
+	moduleExecutionFlow
+Calling init for 'moduleA listener'
+Calling init for 'current module listener'
+Calling start for 'moduleA listener'
+Calling start for 'current module listener'
+		[pass] main_test1
+
+
+		1 passing
+		0 failing
+		0 skipped
+Calling stop for 'current module listener'
+Calling stop for 'moduleA listener'
+
+	moduleExecutionFlow.moduleA
+Calling init for 'moduleA listener'
+Calling start for 'moduleA listener'
+		[pass] test1
+
+
+		1 passing
+		0 failing
+		0 skipped
+Calling stop for 'moduleA listener'
```

Comment:
Shall we add a new line at the end of file?

Reply:
Since the whole content of this text file has been used to assert the linux output, adding a newline will lead to test failure.

Reply:
Can't we fix this from the assertion part?

Reply:
Added a fix to remove leading and trailing white spaces

Reply:
There is an inconsistency/are different behaviors in printing the test results in the console as well. A trailing new line has been added to the console results for the test runs which generating test report. 
```
Generating Test Report
	runtime-api-tests/target/report/test_results.json

```
For other cases no trailing newlines in the console results. This newline has been added always after the cases like above, to keep a line space between them and next line entries, as follows 
```
Generating Test Report
	function-mocking-tests/target/report/test_results.json

error: there are test failures
```
Anyway, no need for a new line after the following message in the console result.
```
error: there are test failures
```
If we need to have trailing new line character in the console results text files, we need to strip the trailing lines of the both test results and the assertion test file contents to make the assertion success.